google/cloud/aiplatform/utils/column_transformations_utils.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud.aiplatform.datasets.column_names_dataset import _ColumnNamesDataset
+from typing import Dict, List, Optional, Tuple
```

Comment:
Python std lib import here and `warnings` below should be imported as a separate block.

Reply:
Fixed